docs/src/tutorials/tutorial_4.md

```diff
@@ -26,6 +26,7 @@ From this tutorial, you will learn how to:
 ## Starting with the Basics of SubAction
 
 The `ground` function should be familiar to you as well as the general structure of the `javis` function if you have seen the first [tutorial](tutorial_1.md).
+In this tutorial, rather than calling the `javis` function by itself, we are going to be calling it from the function we are creating to create our mascot, `face`: 
```

Comment:
Maybe call `face()` in the final code snippet such that copy pasting it in the REPL will actually work.